### PR TITLE
Updates for response page styling 

### PR DESCRIPTION
**Pull request by lgalis**
enable asset pipeline

Screenshot from 2020-02-10 14-05-29

### CODE REVIEW

File: app/controllers/api/v1/stageaction/result.html.erb [hunk 1]
```diff
@@ -51,4 +51,5 @@
       </div>
     </section> 
   </section>
+</main>
```
Comment (epwinchell) [inline app/controllers/api/v1/stageaction/result.html.erb]: @lgalis The "main" tag for both show and result is closed in the footer. This need needs to be removed.

**Comment by epwinchell** [comment]
@bzwei Tested works great

**Comment by bzwei** [comment]
Thank both of you 💯

**Comment by gmcculloug** [comment]
@epwinchell does this resolve issue https://github.com/RedHatInsights/approval-api/issues/268 "Improve layout for mobile"?

**Comment by epwinchell** [comment]
@gmcculloug No. It's unrelated. I will have follow-up PRs.

**Pull request merged**